src/stdlib.z80

```diff
@@ -8,31 +8,18 @@ section "bank 0 stdlib.z80", rom0
 ;;
 ; Copies BC bytes from HL to DE.
```

Comment:
Sorry, I forgot to update the comments here, the direction of movement goes from DE to HL